quidel_covidtest/delphi_quidel_covidtest/run.py

```diff
@@ -123,53 +167,70 @@ def run_module(params: Dict[str, Any]):
                          wip_signal=params["indicator"]["wip_signal"],
                          prefix="wip_")
     smoothers = get_smooth_info(sensors, SMOOTHERS)
-    for geo_res in NONPARENT_GEO_RESOLUTIONS:
-        geo_data, res_key = geo_map(geo_res, data)
-        geo_groups = geo_data.groupby(res_key)
-        for agegroup in AGE_GROUPS:
-            for sensor in sensors:
-                if agegroup == "total":
-                    sensor_name = sensor
-                else:
-                    sensor_name = "_".join([sensor, agegroup])
-                logger.info("Generating signal and exporting to CSV",
-                            geo_res=geo_res,
-                            sensor=sensor_name)
-                state_df = generate_sensor_for_nonparent_geo(
-                    geo_groups, res_key, smooth=smoothers[sensor][1],
-                    device=smoothers[sensor][0], first_date=first_date,
-                    last_date=last_date, suffix=agegroup)
-                dates = create_export_csv(
-                    state_df,
-                    geo_res=geo_res,
-                    sensor=sensor_name,
-                    export_dir=export_dir,
-                    start_date=export_start_date,
-                    end_date=export_end_date)
-                if len(dates) > 0:
-                    stats.append((max(dates), len(dates)))
-    assert geo_res == "state" # Make sure geo_groups is for state level
-    # County/HRR/MSA level
-    for geo_res in PARENT_GEO_RESOLUTIONS:
-        geo_data, res_key = geo_map(geo_res, data)
-        for agegroup in AGE_GROUPS:
-            for sensor in sensors:
-                if agegroup == "total":
-                    sensor_name = sensor
-                else:
-                    sensor_name = "_".join([sensor, agegroup])
-                logger.info("Generating signal and exporting to CSV",
-                            geo_res=geo_res,
-                            sensor=sensor_name)
-                res_df = generate_sensor_for_parent_geo(
-                    geo_groups, geo_data, res_key, smooth=smoothers[sensor][1],
-                    device=smoothers[sensor][0], first_date=first_date,
-                    last_date=last_date, suffix=agegroup)
-                dates = create_export_csv(res_df, geo_res=geo_res,
-                                          sensor=sensor_name, export_dir=export_dir,
-                                          start_date=export_start_date,
-                                          end_date=export_end_date,
-                                          remove_null_samples=True)
+    n_cpu = min(8, cpu_count()) # for parallelization
+    with Manager() as manager:
+         # for using loggers in multiple threads
+        lock = manager.Lock() # pylint: disable=no-member
```

Comment:
This is a bug where pylint claims `Manager` doesn't have that method. [It's fixed in a dev branch.](https://github.com/pylint-dev/pylint/issues/3313)

Reply:
its probably worth noting that fix reference in a comment, and/or maybe even adding a [pylint `useless-suppression`](https://pylint.pycqa.org/en/stable/user_guide/messages/information/useless-suppression.html) so that this shows up and can be removed when its actually fixed (maybe by #1886 )

Reply:
I added a comment about this! Didn't enable `useless-suppression`, because that causes a very silly loop - initially linting fails with:
```
delphi_quidel_covidtest/run.py:175:0: I0021: Useless suppression of 'no-member' (useless-suppression)
```
But when the `pylint: disable=no-member` comment is removed, linting fails with:
```
delphi_quidel_covidtest/run.py:174:15: E1101: Instance of 'SyncManager' has no 'Lock' member; maybe 'RLock'? (no-member)
```